content/docs/vmclarity/getting-started/deploy-docker/_index.md

```diff
@@ -0,0 +1,74 @@
+    ```
+
+1. Create a new directory, extract the files and navigate to the directory.
+
+    ```shell
+    mkdir docker-compose-v{{< param "latest_version" >}}
+    tar -xvzf docker-compose-v{{< param "latest_version" >}}.tar.gz -C docker-compose-v{{< param "latest_version" >}}
+    cd docker-compose-v{{< param "latest_version" >}}
+    ```
+
+1. Start every control plane element with the docker compose file.
+
+    ```shell
+    docker compose --project-name vmclarity --file docker-compose.yml up -d --wait --remove-orphans
+    ```
+
+    The output should be similar to:
+
+    ```
+    [+] Running 14/14
+    ⠿ Network vmclarity                        Created                                                       0.2s
+    ⠿ Volume "vmclarity_grype-server-db"       Created                                                       0.0s
+    ⠿ Volume "vmclarity_apiserver-db-data"     Created                                                       0.0s
+    ⠿ Container vmclarity-orchestrator-1       Healthy                                                      69.7s
+    ⠿ Container vmclarity-yara-rule-server-1   Healthy                                                      17.6s
+    ⠿ Container vmclarity-exploit-db-server-1  Healthy                                                      17.7s
+    ⠿ Container vmclarity-swagger-ui-1         Healthy                                                       7.8s
+    ⠿ Container vmclarity-trivy-server-1       Healthy                                                      26.7s
+    ⠿ Container vmclarity-uibackend-1          Healthy                                                      17.6s
+    ⠿ Container vmclarity-ui-1                 Healthy                                                       7.7s
+    ⠿ Container vmclarity-freshclam-mirror-1   Healthy                                                       7.8s
+    ⠿ Container vmclarity-grype-server-1       Healthy                                                      37.3s
+    ⠿ Container vmclarity-gateway-1            Healthy                                                       7.7s
+    ⠿ Container vmclarity-apiserver-1          Healthy                                                      17.7s
+    ```
+
+    Please note that the `image_override.env` file enables you to use the images you build yourself. You can override parameters in the `docker-compose.yml` by passing a custom env file to the `docker compose up` command via the `--env-file` flag. The `/installation/docker/image_override.env` file contains an example overriding all the container images.
+
+1. Check the running containers in the Docker desktop.
+
+    <p align="center" width="100%">
+        <img width="75%" src="vmclarity-docker.png">
+    </p>
+
+1. Open the VMClarity UI in your browser at [http://localhost:80/](http://localhost:80/).
```

Comment:
How about adding a header before this part where we start accessing the UI? `## Access VMClarity UI`

Reply:
done!